src/components/Chatbot/Chatbot.tsx

```diff
@@ -23,46 +19,115 @@ const NewConversationButton = styled(PrimaryButton)(() => ({
 }));
 
 export const Chatbot = () => {
-  const [conversation, setConversation] = useState<Message[]>([]);
+  const [conversation, setConversation] = useState<MessageBubbleProps[]>([]);
```

Comment:
I find it a little confusing that you call the messages "conversation". "conversation" is an object which has `created_at` and `messages`.

Reply:
changed to messages